strategy.json

```diff
@@ -5,7 +5,7 @@
     "include": [
       {
         "os": "ubuntu-20.04",
-        "python": "3.6"
+        "python": "3.7"
```

Comment:
We need to continue testing on Python 3.6 because we support RHEL 8 in ROS 2 which uses it, despite being EOL.